data/pages/career.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Karriere
```

Comment:
the `onCreateNode` approach I uses requires some meta data for the markdown file. This title is currently only used fro the preview images but I think it would be good to use it for the actual headline too? It's otherwise a pretty inconsistent markdown handling.

Reply:
Definitely!